Lib/glyphsLib/parser.py

```diff
@@ -278,9 +288,19 @@ def dumps(obj):
     dump(obj, fp, **kwargs)
     return fp.getvalue()
 
-def main(args=None):
-    for arg in args:
-        dump(load(open(arg, 'r', encoding='utf-8')), sys.stdout)
+
+def _load_dump(filenames):
```

Comment:
this `_load_dump` function seems unused

Reply:
Yeah.  Left it because perhaps we want to test this again someday.